src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -155,6 +155,12 @@
     "NAME": "Determination",
     "DESCRIPTION": "Remember stsjorbsmod:Pride. NL At the end of this turn, stsjorbsmod:Snap. NL Exhaust.",
     "UPGRADE_DESCRIPTION": "Remember stsjorbsmod:Pride. NL Exhaust."
+  },"stsjorbsmod:Discordance": {
+  "NAME": "Discordance",
+  "DESCRIPTION": "Deal damage equal to the number of card types in your hand times !M!.",
```

Comment:
I worry that the term "card type" might not be intuitively clear to a player; I can see it being misinterpreted as "different card names". We should maybe make a `Card\u00a0Type` keyword that clarifies "Card Type means Attack, Skill, Power, Curse, or Status" or something

Reply:
Added Card Type/s tooltip